src/re2parser.cc

```diff
@@ -25,6 +25,9 @@
 #include <re2/re2/prog.h>
 #include <re2/util/logging.h>
 
+// Will be used as epsilon symbol on transitions
+#define EPSILON 306
```

Comment:
We agreed that I use some random numbers that do not collide with standard characters representation for special characters. Numbers representing special characters start at 300 and go up (for example beginning of the line is currently represented by 300). This will be changed when the final representation is decided. So it is just a temporal solution. However, I noticed that I skipped 305.